### DISCUSSION TITLE
Feature Requests Master Discussion

### DISCUSSION THREAD

**Discussion by algertc** [algertc/ALPR-Database#28] [Announcements]
Please comment any feature requests here. I am still actively developing this so any input is appreciated. I have Implemented most of the recent feature requests. I will push a live update update (non versioned) tonight, but please do submit any issue or IPCamTalk post for anything you find missing. I'm currently working on TPMS detection which, yes, is an additional effort for most, but will provide an enormous increase in detection capability.

**Comment by kummerr** [comment]
So has the live update been pushed?

**Reply by MikeLud** [reply]
The new update was pushed today.

**Comment by olympia** [comment]
It would be nice if we could add plates to the known plates database or the watchlist manually (I mean even if the given plate was not showing up yet).

**Reply by algertc** [reply]
> It would be nice if we could add plates to the known plates database or the watchlist manually (I mean even if the given plate was not showing up yet).

I added this a a couple weeks ago. The full update is coming soon, but this should be available in the nightly version. Please try the install script and select the nightly option and see if it works if you haven't updated yet.

**Comment by prsmith777** [comment]
I would like to request ability to have ALPR program send an MQTT message when a plate is received that contains the plate name as the payload. Not the plate number but the learned/saved name I have associated with a certain plate that is in the database. 

For instance, let's say "ABC 123" plate has a saved name as "Bob's Cadillac" in the database.  When Bob drives up the driveway with plate ABC 123, MQTT message is sent with payload "Bob's Cadillac". 

This will help for making announcements with Home Assistant or Home Seer.

**Reply by algertc** [reply]
Thank you. I meant the roadmap here: https://alprdatabase.featurebase.app/

I added it for you: https://alprdatabase.featurebase.app/p/mqtt-notifications-for-ha

**Comment by Skym-IT** [comment]
Soo here's an idea I have as someone who has multiple cameras between multiple locations feeding plates into the database you should use that area chart thing to kind of replicate the time distribution chart but use the different colors to represent different cameras 🤷🏻‍♀️ or something like that it would be nice to be able to narrow it down to each location I have

**Reply by algertc** [reply]
I like it. I can add a switch that shows the same thing for tags too. Thank you.

**Comment by Skym-IT** [comment]
Thank you so much for all your work on this project!On Feb 13, 2025, at 8:34 AM, Charlie Algert ***@***.***> wrote:﻿
I like it. I can add a switch that shows the same thing for tags too. Thank you.

**Comment by thtech182** [comment]
Feature request: statistics on a particular plate similar to the dashboard. Reads by time of day, day of week, camera, etc. That way you can better investigate a particular plate.. always see this one at 0300 on Sundays, must be the paper guy.

**Reply by algertc** [reply]
Try clicking the plate number in the database tab. It should open something just like that on the side. 

Coming back to fix the install problem soon…

**Reply by thtech182** [reply]
Well damn! That's exactly what I was looking for and it was already there!

This is a phenomenal application!
